index.html

```diff
@@ -30,7 +31,77 @@
   </section>
-    
-    
+</div>
+   
+
+  <div id="section1" class="section_2">
+    <img id="snap" src="assets/images/Snapshoot_Portfolio.svg" alt="Snapshot of portfolio">
+    <img id="tonic" src="assets/images/tonic.png" alt="Tonic Project name">
+    <p class="select">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <img class="tags" src="assets/images/Tags.png" alt="Tag buttons">
+    <button class="btn1" type="button">See project</button>
+  </div>
+
+  <section id="section2" class="section_2">
+    <img class="snap2" src="assets/images/Snap2.svg" alt="Snapshot of portfolio">
+    <img class="stories" src="assets/images/stories.svg" alt="Tonic Project name">
+    <p class="select">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <img class="tags" src="assets/images/Tags.png" alt="Tag buttons">
+    <button class="btn1" type="button">See project</button>
+  </section>
+
+  <section id="section3" class="section_2">
+    <img class="snap2" src="assets/images/tonic2.svg" alt="Snapshot of portfolio">
+    <img id="second_tonic" src="assets/images/tonic.png" alt="Tonic Project name">
+    <p class="select">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <img class="tags" src="assets/images/Tags.png" alt="Tag buttons">
+    <button class="btn1" type="button">See project</button>
+  </section>
+
+  <section id="section4" class="section_2">
+    <img class="snap2" src="assets/images/stories2.svg" alt="Snapshot of portfolio">
+    <img class="stories" src="assets/images/stories.svg" alt="Tonic Project name">
+    <p class="select">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <img class="tags" src="assets/images/Tags.png" alt="Tag buttons">
+    <button class="btn1" type="button">See project</button>
+  </section>
+
+  <section id="about-me" class="about_me"> 
+    <div id="greeter"> <h1>About <br>myself</h1> </div>
+    <br> <br>
+    <p id="myself-text">I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+    </p>
+    <p id="connects">LET’S CONNECT</p>
+    <img id="sm_icons" src="assets/images/social-media-icons.svg" alt="social-media icons">
+    <button id="btn2" type="button">Get my resume</button>
+    <p id="lang">Languages</p>
+   <ul class="lang-list">
+      <li><img src="assets/images/js.svg" alt="javascript logo"></li>
+      <li><img src="assets/images/html.svg" alt="html logo"></li>
+      <li><img src="assets/images/css.svg" alt="css logo"></li>
+      <li><img src="assets/images/framework.svg" alt="framework text"></li>
+      <li><img src="assets/images/skills.svg" alt="skills text"></li>
+    </ul> 
```

Comment:
- [ ] Great job adding the technologies inside a list , kindly remove the images since those are titles and links so they need to be HTML codes and not images

Reply:
- [ ] Please, 🙏🏽 complete the requirement changes from the previous code reviewer ☝🏽